docs/setup/plugins/github.md

```diff
@@ -2,4 +2,223 @@
 title: Github
 ---
 
-TODO
+# GitHub Plugin
+
+The GitHub plugin assigns Cred to contributors participating
+on GitHub, for example by writing or reviewing code, filing and triaging bug
+reports, maintaining the build, and cutting new releases.
+
+The plugin does this by downloading extensive data on the repository history
+from GitHub, and using it to create a contribution graph. You can read below to
+see the kinds of nodes and edges in that graph. The data is temporarily cached
+locally, but the source of truth for the GitHub plugin is always GitHub's live
+servers. That means that if content is deleted on GitHub, it will also
+disappear from the GitHub plugin (after a cache refresh).
+
+## Status and Caveats
+
+The GitHub plugin is still in beta. It tends to assign reasonable Cred scores
+to contributors. However, it has difficulty assigning Cred scores to Pull
+Requests (PRs). This is because, in general, the plugin mints Cred based on raw
+activity. Depending on weight choices, every pull will mint a fixed amount of
+Cred, likewise for every comment, issue, review, and so forth.
+
+This is problematic because, to put it simply, not all activity is equally
+valuable. Minting Cred directly on raw activity encourages contributors to
+focus on quantity over quality. As an example, a poorly thought out PR
+ (which requires lots of review and feedback to fix) will generate more
+Cred than a clean, elegant pull request (which merges with a minimum of fuss).
+
+We intend to improve the robustness of the GitHub plugin by
+adding better heuristics for assigning Cred, e.g.:
+- Minting Cred when PRs are merged, rather than when they are created.
+- Allowing custom labels that influence the Cred minted to PRs.
+- Modifying Cred minting based on metrics like the size of the PR.
```

Comment:
We  can take away the first bullet, since that's been fixed. Let's keep the next two.

Reply:
Removed first bullet from both lists, as both addressed minting on merge